content/pt-br/docs/reference/using-api/client-libraries.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Bibliotecas de clientes
+reviewers:
+- ahmetb
+content_type: conceito
+weight: 30
+---
+
+<!-- overview -->
+Esta página contém uma visão geral das bibliotecas clientes para usar a API Kubernetes de várias linguagens de programação.
```

Comment:
* `API Kubernetes` -> `API do Kubernetes`
* `De várias linguagens...` can be `por meio de várias`, what you think?